README.md

```diff
@@ -1,2 +1,18 @@
 # flytectl
 Flyte CLI
+
+In order to dev, needing to work with a cluster.
+
+A local one can be setup via --> https://lyft.github.io/flyte/administrator/install/getting_started.html#getting-started
+
+
+get the gRPC port:
```

Comment:
this is required if Flyte is running in a VPC which is not accessible right?
Otherwise we in sandbox today (locally) we set-up a nodeport service on `locahost:30081`? So maybe we want to clarify this point?

Something like
1. If you have FlyteAdmin hosted behind a load-balancer then - use the DNS 
2. If you are running the standard getting-started sandbox setup locally on docker desktop then - use localhost:30081
3. if you are running sandbox on minikube then run `minikube ip`:30081?
4. if it is running on some k8s cluster not exposed via a LB, then 
your command?

Reply:
Austin had a problem using the nodeport for GRPC so this was the workaround we did to make it work. Let's actually debug that issue and figure out the proper cause... This can live as a workaround still (in a troubleshooting subsection perhaps?)

Reply:
Yes, good to find the underlying issue -- wanted to surface a workaround, in case others experience same